docs/source/run/chat.rst

```diff
@@ -0,0 +1,9 @@
+Join the community
+==================
+
+For questions or support, feel free to join the HiPACE++ chat on the Mattermost platform. This will take a few steps:
```

Comment:
```suggestion
For questions or support, feel free to join the `HiPACE++ chat <https://mattermost.hzdr.de/hipace>`__ on the Mattermost platform. For your first connection, this will take a few steps:
```